docs/background/velocities.rst

```diff
@@ -0,0 +1,787 @@
+the Earth's orbital specific angular momentum vector :math:`\vec{h}_\oplus`.
+It is given by
+
+.. math::
+    
+    i_\oplus = \beta_\mathrm{p} + 90^\circ.
+
+The restriction on the pulsar's ecliptic longitude :math:`-90^\circ \le
+\beta_\mathrm{p} \le 90^\circ` leads to the expected range of allowed
+inclinations :math:`0^\circ \le i_\oplus \le 180^\circ`. The convention for the
+sense of rotation is also the same: :math:`i_\oplus < 90^\circ` for
+counterclockwise rotation when viewing in the :math:`\hat{z}` direction and
+:math:`i_\oplus > 90^\circ` for clockwise rotation.
+
+Earth's ascending node with respect to the line of sight is the point on the
+orbit where Earth passes through the observing plane in the direction of the
+pulsar. In this context, the longitude of ascending node :math:`\Omega_\oplus`
+is equivalent to the position angle of Earth's ascending node with respect to
+the coordinates of the pulsar system:
+
+.. math::
+
+    \Omega_\oplus = \mathcal{P}( X_\mathrm{p}, X_\mathrm{asc,\oplus} ).
+
+Here, :math:`\mathcal{P}( X_1, X_2 )` yields the position angle (east of north)
+from position :math:`X_1` to position :math:`X_2` (for details on this
+computation, see, e.g., the `Wikipedia article on position angle
+<https://en.wikipedia.org/wiki/Position_angle>`_), :math:`X_\mathrm{p} =
+(\alpha_\mathrm{p}, \delta_\mathrm{p})` denotes the equatorial coordinates of
+the pulsar system, and :math:`X_\mathrm{asc,\oplus} =
+(\alpha_\mathrm{asc,\oplus}, \delta_\mathrm{asc,\oplus})` is the equatorial
+coordinates of Earth's ascending node. The latter can be found from its
+ecliptic coordinates :math:`(\lambda_\mathrm{asc,\oplus},
+\beta_\mathrm{asc,\oplus}) = (\lambda_\mathrm{p} - 90^\circ, 0)`.
+
+.. TODO: [maybe include figure here showing Omega_earth]
+
+Finally, under the simplifying assumption that Earth's orbit is circular,
+the time of Earth's passage through the ascending node is given by
+
+.. math::
+    
+    t_\mathrm{asc,\oplus} = t_\mathrm{eqx} + P_\mathrm{orb,\oplus}
+        \frac{ \lambda_\mathrm{asc,\oplus} }{ 360^\circ },
+
+where :math:`t_\mathrm{eqx}` is the time of the March equinox and
+:math:`\lambda_\mathrm{asc,\oplus} = \lambda_\mathrm{p} - 90^\circ` is the
+ecliptic longitude of Earth's ascending node.
+
+
+Combining the pulsar, lens, and Earth terms
+===========================================
+
+Combining the different terms in equation :math:`\ref{eq_v_eff}` contributing
+to :math:`v_\mathrm{eff,\parallel}` gives
+
+.. math::
+
+    v_\mathrm{eff,\parallel} =
+        \underbrace{
+            \frac{ 1 }{ s } v_\mathrm{lens,\parallel}
+        }_\textrm{lens motion}
+      - \underbrace{
+            \frac{ 1 - s }{ s } d_\mathrm{p}
+                \left[ \mu_\mathrm{p,sys,\alpha\ast} \sin( \xi )
+                     + \mu_\mathrm{p,sys,\delta} \cos( \xi )
+                \right]
+        }_\textrm{pulsar's systemic motion}
+      + \underbrace{
```

Comment:
I'm a bit confused why this is not negative, like the systemic velocity term (same for earth orbit)

Reply:
There's a minus sign in front of the v_0 factor, first introduced in the section "The orbit’s orientation on the sky and the sky-plane velocity". Is that what you were missing or is there a deeper problem?

Reply:
I was mostly wondering about the choice. I would have said `v_psr = v_sys+v_orb`. Since here one has `-v_psr`, I would thus have expected minus signs in front of both. I think it would be better to change it here and in the "orbit's orientation" section...

Reply:
Alright, we can give the terms corresponding to `v_p_sys` and `v_p_orb` in the equation for `v_eff` the same sign by changing the definition of `delta_omega` to `delta_omega = omega - xi` (it's currently the other way round: `delta_omega = xi - omega`). It will require a bit of an overhaul (also in the other tutorial documents, to make them consistent), but I think it's a good idea, so I'll implement this.

Regarding the definition of `delta_omega_p` and `delta_omega_e`, it makes a bit more sense to me anyway to introduce angles that give the orientation of the pulsar's and Earth's orbit with respect to the screen, rather than introduce angles that give the orientation of the (single) screen with respect to the two different orbits.

Another consequence of this change will be that the final equation for the "scaled effective velocity" will have the form
`| C - A_p sin(phi_p - chi_p) - A_e sin(phi_e - chi_e) |`
rather than a sum of only positive terms. That would not happen, however, if we also flip the sign in the definition of the effective velocity, i.e. define `v_eff` as
`v_eff = (1-s)/s v_psr - 1/s v_lens + v_earth`

Reply:
@mhvk - It's now ready for another review